compiler/rustc_expand/src/build.rs

```diff
@@ -36,7 +36,7 @@ impl<'a> ExtCtxt<'a> {
         );
         let args = if !args.is_empty() {
             let args = args.into_iter().map(ast::AngleBracketedArg::Arg).collect();
-            ast::AngleBracketedArgs { args, span }.into()
+            Some(ast::AngleBracketedArgs { args, span }.into())
```

Comment:
Why is the `Some` needed now?

Reply:
Because I changed the `Into<P<GenericArgs>>` impl, since it was never not returning `Some`, it's clearer to wrap it in `Some` at the usage site